lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb

```diff
@@ -52,6 +48,8 @@ def breadcrumbs
       def preferred_taxon
         if preferred_priority_taxon
           priority_taxons.find { |t| t["content_id"] == preferred_priority_taxon }
+        elsif tagged_to_both_brexit_child_taxons?
```

Comment:
this is cool, so we could use the `preferred_priority_taxon` approach if we wanted one of the taxons to appear in some circumstances (if they were tagged to both).